agent/ecscni/types_linux.go

```diff
@@ -28,6 +28,9 @@ const (
 	// defaultAppMeshIfName is the default name of app mesh to setup iptable rules
 	// for app mesh container. IfName is mandatory field to invoke CNI plugin.
 	defaultAppMeshIfName = "aws-appmesh"
+	// defaultServiceConnectIfName is the default ifname used for invoking SC CNI plugin.
+	// IfName is a mandatory field to invoke any CNI plugin. But it's not actually used in this case.
+	defaultServiceConnectIfName = "ecs-serviceconnect"
```

Comment:
Why cant we actually send the interface name available to this container? The interface name is not used in awsvpc but it would be for bridge.

Reply:
I saw the CNI plugin CR using `eth0` (which indeed is the default ifname for docker bridge netns) so I didn't add anything here. I can add it.

Reply:
Yes I changed it to use `ifName` in the latest revision.

Reply:
Cool, will address in next revision.